cas-server-documentation/installation/OAuth-OpenId-Authentication.md

```diff
@@ -95,15 +28,18 @@ Every OAuth client must be defined as a CAS service (notice the new *clientId* a
 <bean id="serviceRegistryDao" class="org.jasig.cas.services.InMemoryServiceRegistryDaoImpl">
```

Comment:
Shouldn't we have our documentation based on JSON examples right now? As the default services registry is the JSON one?